LICENSE.md

```diff
@@ -0,0 +1,80 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved. "Business Source License" is a trademark of
+MariaDB Corporation Ab.
+
+---
+
+Parameters
+
+Licensor: Sablier Labs Ltd
+
+Licensed Work: Sablier Airdrops The Licensed Work is (C) 2024 Sablier Labs Ltd
+
+Additional Use Grant: Any uses listed and defined at license-grants.sablier.eth
+
+Change Date: The earlier of 2029-01-15 or a date specified at airdrops-license-date.sablier.eth
```

Comment:
Did you change your mind about this @PaulRBerg? You said to use the same ens across lockup, airdrops and flow.

https://sablier.slack.com/archives/C06BB90SY02/p1731733224286589?thread_ts=1731718257.697379&cid=C06BB90SY02

Reply:
I didn't change my mind.

We will use the same ENS subdomain for the license grants (line 14).

However, we have to use different subdomains for the license dates (line 16).

Anyway, the latter are not even registered. We would register them only if we want to release the code earlier than 2029-01-15.